docs/source/technical_tutorials/visualization/visualize_lonboard.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>should this be a code cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>